onnx/defs/schema.h

```diff
@@ -495,6 +495,31 @@ class OpSchemaRegistry {
                   << schema.file() << " line " << schema.line();
         abort();
       }
+
+      auto ver_range_map = DomainToVersionRange::Instance().Map();
```

Comment:
There's also another kind of mistake. 

Developer bumps operator version without checking the domain max version here. Say, max version is 3 here, the specific op version is 1, developer may bump the version to 2 by mistake :).

Agree with you, we may need bot to do this bumping.